oidc_provider/tests/cases/test_authorize_endpoint.py

```diff
@@ -276,14 +276,14 @@ def test_response_uri_is_properly_constructed(self):
         parsed = urlsplit(response['Location'])
         params = parse_qs(parsed.query or parsed.fragment)
         state = params['state'][0]
-        self.assertEquals(self.state, state, msg="State returned is invalid or missing")
+        self.assertEqual(self.state, state, msg="State returned is invalid or missing")
```

Comment:
We used to have a lot of these but have converted them to bare asserts overt time. e.g could do `assert self.state = params['state'][0]`.  maybe worth doing here with the other cleanup!